.github/workflows/test-and-deploy.yml

```diff
@@ -331,12 +331,21 @@ jobs:
           if-no-files-found: error
           retention-days: 7
 
+      - name: Set asset name
+        run: |
+          if [[ ${{ matrix.runner }} == 'pi5-builder' ]]; then
+            echo "ASSET_NAME_SUFFIX=-pi5" >> $GITHUB_ENV
+          elif [[ ${{ matrix.os }} == 'bullseye' ]]; then
```

Comment:
same here: `elif [[ ${{ matrix.os }} == *'bullseye'* ]]; then` ?

Reply:
The OS matrix name os bullseye, there is no need to regex.

Reply:
Sure